tyson/README.md

```diff
@@ -90,29 +94,32 @@ Simply configure your editor to treat `.tson` files as TypeScript files.
 
 
 ## Why?
+
 Almost all developer tools require some form of configuration. In our opinion,
 an ideal configuration language should be:
-+ **Easy to read and write by humans**
-+ **Easy to parse and generate by machines**
-+ **Type safe** - so that it's easy to validate the output
-+ **Programmable** – so that you can abstract complex configuration patterns
+
+- **Easy to read and write by humans**
+- **Easy to parse and generate by machines**
+- **Type safe** - so that it's easy to validate the output
+- **Programmable** – so that you can abstract complex configuration patterns
   into reusable functions
-+ **Secure** - if we want programmable configuration, its execution should
+- **Secure** - if we want programmable configuration, its execution should
   not affect the application that loads it.
-+ **Have a well-understood syntax** - without major gotchas that can result in errors
-+ **Based on a widely used standard** – nobody wants to have to learn a new
+- **Have a well-understood syntax** - without major gotchas that can result in errors
+- **Based on a widely used standard** – nobody wants to have to learn a new
   language just to configure a tool
-+ **Easy to migrate to** - tools that already use JSON for configuration should
+- **Easy to migrate to** - tools that already use JSON for configuration should
   be able to gradually adopt the new language, while retaining compatibility
   with existing JSON configuration files.
 
 Traditionally, the most popular choices for configuration have been: JSON, YAML
 or TOML, but they each have drawbacks:
-+ **JSON**: doesn't support comments, trailing commas, or multi-line strings.
-+ **YAML**: has an ambigous syntax. For example the token `no` is interpreted
+
+- **JSON**: doesn't support comments, trailing commas, or multi-line strings.
+- **YAML**: has an ambigous syntax. For example the token `no` is interpreted
   as a boolean, often in cases where you want it to be a string. See
   https://noyaml.com/ for more examples.
-+ **TOML**: Gets unwidely when there's multiple levels of nesting.
+- **TOML**: Gets unwidely when there's multiple levels of nesting.
```

Comment:
What's the difference between `+` and `-`?

Reply:
I think `*` or `-` are the more standard Markdown bullets